### ISSUE TITLE
Click bug with Dragon NaturallySpeaking 

### ISSUE THREAD

**Issue by lizcameron** [bbc/psammead#2619]
**Describe the bug**
Dragon NaturallySpeaking doesn't work as expected on the media block, article page or front page. It can navigate through the page but does not respond to clicking on links or buttons, e.g. play. Mixed results have been found when testing this, with some users able to click play but unable to then control further, e.g. pause or stop. More investigation is needed to determine if there is a problem with Simorgh pages more widely, or just a specific block/page, and what the problem is. 

**To Reproduce**
Steps to reproduce the behaviour:

1. Open Dragon NaturallySpeaking and set up (if required)
2. Open IE
3. Go to https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo
4. Use Dragon NaturallySpeaking to locate the media block
5. Attempt to play the video in the media block
6. See error

**Expected behaviour**
Outlined here https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-dragon

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 7
- Browser: IE 11
- v13.0

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
